ajax/libs/paver/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "paver",
+  "filename": "js/jquery.paver.min.js",
+  "version": "1.3.3",
+  "description": "A minimal panorama/image viewer replicating the effect seen in Facebook Pages app.",
+  "homepage": "https://terrymun.github.io/paver/",
```

Comment:
@PeterDaveHello The homepage will redirect the url to `https://terrymun.github.io/paver/demo/index.html`. Is it necessary to change the url?